monorepo-builder.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\MonorepoBuilder\ValueObject\Option;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $parameters = $containerConfigurator->parameters();
+
+    $parameters->set(Option::DATA_TO_APPEND, [
+        'require-dev' => [
+            'symplify/monorepo-builder' => '^8.2',
+            'symplify/changelog-linker' => '^8.2',
+            'symplify/easy-coding-standard' => '^8.2',
+            'phpstan/phpstan' => '^0.12.42',
+            'sensiolabs/security-checker' => '^5.0',
+        ],
+    ]);
+
+    $parameters->set(Option::DIRECTORIES_TO_REPOSITORIES, [
+        // tip for lazier setup with mask: https://github.com/symplify/monorepo-builder#5-split-directories-to-git-repositories
+        // 'packages/*' => 'git@github.com:eonx-com/*.git',
```

Comment:
@natpage Just checking if you noticed this. Extra lazy sauce 😜

Reply:
You need to rename all directories here to lowercase ... Well, I can send PR a actually there is no other blocker